### PR TITLE
Fixing windows pr job by added repo ref for cloud-provider-azure

### PR DESCRIPTION
**Pull request by marosset**
/sig windows

### CODE REVIEW

File: config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml [hunk 1]
```diff
@@ -166,7 +171,7 @@ presubmits:
     extra_refs:
     - org: kubernetes-sigs
       repo: cluster-api-provider-azure
-      base_ref: main
+      base_ref: release-1.8
```
Comment (marosset) [inline config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml:174]: We need to get https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/3388 into release-1.8

Comment (CecileRobertMichon) [reply config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml:174]: added a hold so we can get that in, feel free to remove the hold when ready

**Pull request merged**